modules/fx/arrow-fx/src/main/kotlin/arrow/fx/IO.kt

```diff
@@ -47,8 +47,10 @@ inline fun <A> IOOf<A>.fix(): IO<A> =
 typealias IOProc<A> = ((Either<Throwable, A>) -> Unit) -> Unit
 typealias IOProcF<A> = ((Either<Throwable, A>) -> Unit) -> IOOf<Unit>
 
+typealias IO<A> = BIO<Nothing, A>
```

Comment:
`IO<A>` is `Throwable` for compatibility, right?

Reply:
Yes.
- `IO<A> <~> suspend () -> A`
-  `BIO<E, A> <~> EitherT<ForIO, E, A> <~> suspend () -> Either<E, A>`